init.lua

```diff
@@ -209,293 +209,19 @@ minetest.register_craft({
-	"default:sign_wall_wood",
-	"default:silver_sand",
-	"default:skeleton_key",
-	"default:snow",
-	"default:snowblock",
-	"default:steelblock",
-	"default:steel_ingot",
-	"default:stick",
-	"default:stone",
-	"default:stone_block",
-	"default:stonebrick",
-	"default:stone_with_coal",
-	"default:stone_with_copper",
-	"default:stone_with_diamond",
-	"default:stone_with_gold",
-	"default:stone_with_iron",
-	"default:stone_with_mese",
-	"default:sword_bronze",
-	"default:sword_diamond",
-	"default:sword_mese",
-	"default:sword_steel",
-	"default:sword_stone",
-	"default:sword_wood",
-	"default:torch",
-	"default:tree",
-	"default:water_flowing",
-	"default:water_source",
-	"default:wood",
-	"default:marram_grass_1",
-	"default:marram_grass_2",
-	"default:marram_grass_3",
-	"default:sand_with_kelp",
-	"doors:door_glass",
-	"doors:door_obsidian_glass",
-	"doors:door_steel",
-	"doors:door_wood",
-	"doors:trapdoor",
-	"doors:trapdoor_steel",
-	"dye:black",
-	"dye:blue",
-	"dye:brown",
-	"dye:cyan",
-	"dye:dark_green",
-	"dye:dark_grey",
-	"dye:green",
-	"dye:grey",
-	"dye:magenta",
-	"dye:orange",
-	"dye:pink",
-	"dye:red",
-	"dye:violet",
-	"dye:white",
-	"dye:yellow",
-	"farming:bread",
-	"farming:cotton",
-	"farming:cotton_1",
-	"farming:cotton_2",
-	"farming:cotton_3",
-	"farming:cotton_4",
-	"farming:cotton_5",
-	"farming:cotton_6",
-	"farming:cotton_7",
-	"farming:cotton_8",
-	"farming:flour",
-	"farming:hoe_bronze",
-	"farming:hoe_diamond",
-	"farming:hoe_mese",
-	"farming:hoe_steel",
-	"farming:hoe_stone",
-	"farming:hoe_wood",
-	"farming:seed_cotton",
-	"farming:seed_wheat",
-	"farming:straw",
-	"farming:string",
-	"farming:wheat",
-	"farming:wheat_1",
-	"farming:wheat_2",
-	"farming:wheat_3",
-	"farming:wheat_4",
-	"farming:wheat_5",
-	"farming:wheat_6",
-	"farming:wheat_7",
-	"farming:wheat_8",
-	"fire:basic_flame",
-	"fire:flint_and_steel",
-	"fire:permanent_flame",
-	"flowerpot:empty",
-	"flowers:dandelion_white",
-	"flowers:dandelion_yellow",
-	"flowers:geranium",
-	"flowers:mushroom_brown",
-	"flowers:mushroom_red",
-	"flowers:rose",
-	"flowers:tulip",
-	"flowers:viola",
-	"flowers:waterlily",
-	"flowers:chrysanthemum_green",
-	"flowers:tulip_black",
-	"fireflies:firefly",
-	"fireflies:bug_net",
-	"fireflies:firefly_bottle",
-	"screwdriver:screwdriver",
-	"tnt:gunpowder",
-	"vessels:drinking_glass",
-	"vessels:glass_bottle",
-	"vessels:glass_fragments",
-	"vessels:shelf",
-	"vessels:steel_bottle",
-	"wool:black",
-	"wool:blue",
-	"wool:brown",
-	"wool:cyan",
-	"wool:dark_green",
-	"wool:dark_grey",
-	"wool:green",
-	"wool:grey",
-	"wool:magenta",
-	"wool:orange",
-	"wool:pink",
-	"wool:red",
-	"wool:violet",
-	"wool:white",
-	"wool:yellow",
-	"xpanes:bar_flat",
-	"xpanes:pane_flat",
-	"xpanes:obsidian_pane",
-	"binoculars:binoculars",
-	"map:mapping_kit",
-}) do
-	frame.register(node)
+local items_to_frame = {}
+
+for i, node in pairs(minetest.registered_items) do
+    if minetest.get_item_group(node.name, "not_in_creative_inventory") == 0
```

Comment:
```suggestion
    if (minetest.get_item_group(node.name, "not_in_creative_inventory") == 0
```

Reply:
Missing `)` at the end, otherwise it'll throw error.

Reply:
is the one I suggested line 217 not enough? (I don't know how to make a multi-lines suggestion and this is not very convenient to read)

Reply:
The one on 217 is enough. I copied your changes into my local copy and I don't get any errors from this.